projects/ADuCM3029_demo_cn0428_cn0429/src/adi_m355_gas_sensor.cpp

```diff
@@ -622,21 +604,21 @@ SENSOR_RESULT M355_GAS::RunEISMeasurement()
  * @return 	returns result of I2C communication
  *
  * @details This write command instructs the M355 board to begin returning EIS
- *		data representing Frequency, Magnitude and Phase when the next
- *		read command is issued. Continue reading until 0xFF is read
- *		continuously. Normally 144 bytes are expected. This consists of
- *		12bytes for the 12 frequencies (as detailed in the Run EIS
- *		command). Frequency, Magnitude and Phase are 4 bytes each. So 12
- *		bytes for 12 frequency points = 144bytes. The data returned is
- *		in byte array format representing IEEE754 single precision 32bit
- *		floating point data.
+ *		data representing Frequency, Magnitude, Phase and Complex Magnitude
+ *		(real and imaginary) when the next read command is issued. Continue
+ *		reading until 0xFF is read continuously. Normally 240 bytes are expected.
+ *		This consists of 20 bytes for the 12 frequencies (as detailed in the
+ *		Run EIS	command). Frequency, Magnitude, Phase and Complex Magnitude
+ *		are 4 bytes each. So 20 bytes for 12 frequency points = 240 bytes.
+ *		The data returned is in byte array format representing IEEE754 single
+ *		precision 32bit	floating point data.
  */
 SENSOR_RESULT M355_GAS::ReadEISResults(uint8_t *pSensData)
 {
-	this->I2CReadWrite(READ, READ_EIS_RESULTS, pSensData, 144u);
+	this->I2CReadWrite(READ, READ_EIS_RESULTS, pSensData, 240u);
```

Comment:
Can these values be contained in a define? It would be much easier to change if needed next time.